pkg/config/static/static_config.go

```diff
@@ -0,0 +1,90 @@
+package static
+
+import (
+	httpProvider "github.com/containous/maesh/pkg/proxy/provider/http"
+	traefikStatic "github.com/containous/traefik/v2/pkg/config/static"
+	"github.com/containous/traefik/v2/pkg/ping"
+	"github.com/containous/traefik/v2/pkg/provider/file"
+	"github.com/containous/traefik/v2/pkg/provider/rest"
+	"github.com/containous/traefik/v2/pkg/types"
+)
+
+// DefaultInternalEntryPointName the name of the default internal entry point
+const DefaultInternalEntryPointName = "traefik"
+
+// Configuration is the static configuration
+type Configuration struct {
+	Global *traefikStatic.Global `description:"Global configuration options" json:"global,omitempty" toml:"global,omitempty" yaml:"global,omitempty" export:"true"`
```

Comment:
Why not embed a `traefikStatic.Configuration` instead of duplicating all of its fields here? Are we planning on deleting some or just adding new ones?

Reply:
We are deleting a large portion of this configuration (in particular the providers)

Reply:
I have rethought this, and I might be able to make it work